src/app/components/pipes/file-size.pipe.ts

```diff
@@ -6,12 +6,12 @@ import { Pipe, PipeTransform } from '@angular/core';
 export class FileSizePipe implements PipeTransform {
 
   /**
-   * Return size of file formatted as XXXmb
-   * @param numOfMb
+   * Return size of file formatted as XXX{mb,gb}
+   * @param sizeInBytes -- file size in bytes
    */
-  transform(numOfMb: number): string {
-    if (numOfMb) {
-      const rounded = Math.round(numOfMb);
+  transform(sizeInBytes: number): string {
+    if (sizeInBytes) {
+      const rounded = Math.round(sizeInBytes / 1000000.00);
```

Comment:
This makes the size appear as `0mb` for every video -- on old _vha_ files and on new ones created with this branch.
```suggestion
      const rounded = Math.round(sizeInBytes);
```

Reply:
Since we're now hashing the file name and file size, I'd really like the actual file size to be in the library file, rather than the rounded version.

Especially since 1 calculation on a system that can do 3 billion a second isn't much overhead! ;) Reading in the thumbnail to display already dwarfs any effect this would have.